src/hooks/useHandleOnScrollCallback/index.ts

```diff
@@ -1,25 +1,25 @@
 import React, { useCallback } from 'react';
-import { SCROLL_BUFFER } from '../../../../utils/consts';
+import { SCROLL_BUFFER } from '../../utils/consts';
 
 export interface UseHandleOnScrollCallbackProps {
-  setShowScrollDownButton: React.Dispatch<React.SetStateAction<boolean>>;
   hasMore: boolean;
   onScroll(fn: () => void): void;
   scrollRef: React.RefObject<HTMLDivElement>;
+  setShowScrollDownButton?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Maybe there's a better way to make this logic reusable but not to break anything for now, just made this one to optional.